src/Microsoft.ServiceHub.Framework/DelegatingServiceJsonRpcDescriptor.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.IO.Pipelines;
+using StreamJsonRpc;
+
+namespace Microsoft.ServiceHub.Framework;
+
+/// <summary>
+/// A delegating RPC descriptor for services that support JSON-RPC. By default this descriptor will delegate operations to the wrapped descriptor.
+/// </summary>
+/// <remarks>
+/// This implementation will also ensure inner descriptor is updated with any changes to multiplexing stream options or exception strategy.
+/// </remarks>
+[DebuggerDisplay("{" + nameof(DebuggerDisplay) + ",nq}")]
+public class DelegatingServiceJsonRpcDescriptor : ServiceJsonRpcDescriptor
```

Comment:
Presumably this class is useless as-is. If what it enables requires deriving from it, can we add `abstract` to the class declaration?

Reply:
Will do